docs/rule/element-attributes-order.md

```diff
@@ -0,0 +1,38 @@
+## element-attributes-order
+
+Element attributes must be ordered as:
+
+* arguments
+* attributes
+* element modifiers
+* ...attributes
+* ?attributes
```

Comment:
what is this?

Reply:
`attributes` = html attributes, such as `class="foo"`
`...attributes` = ember/glimmer `...attributes` splat
`?attributes` = same as html attributes, yet optional. It must be allowed to have you own html attributes after the `...attributes` splat to allow overrides.